docs/en/event-daemon/event-daemon-technical-details.md

```diff
@@ -4,3 +4,127 @@ title: Technical Details
 pagename: event-daemon-technical-details
 lang: en
 ---
+
+# Technical Overview
+
+<a id="Event_Types"></a>
+## Event Types
+
+The event types your triggers can register to be notified of are generally respect the following form `Shotgun_[entity_type]_[New|Change|Retirement|Revival]`. Here are a few examples of this pattern:
+
+    Shotgun_Note_New
+    Shotgun_Shot_New
+    Shotgun_Task_Change
+    Shotgun_CustomEntity06_Change
+    Shotgun_Playlist_Retirement
+    Shotgun_Playlist_Revival
+
+Some notable departures from this pattern are used for events that aren't related to entity record activity but rather key points in application behavior.
+
+    CRS_PlaylistShare_Create
+    CRS_PlaylistShare_Revoke 
+    SG_RV_Session_Validate_Success
+    Shotgun_Attachment_View
+    Shotgun_Big_Query
+    Shotgun_NotesApp_Summary_Email
+    Shotgun_User_FailedLogin
+    Shotgun_User_Login
+    Shotgun_User_Logout
+    Toolkit_App_Startup
+    Toolkit_Desktop_ProjectLaunch
+    Toolkit_Desktop_AppLaunch
+    Toolkit_Folders_Create
+    Toolkit_Folders_Delete    
```

Comment:
Nothing to change here, I just wanted to note that leaving them like this instead of using the product include is the right decision for now.